XLD/XLD.download.recipe

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Description</key>
+	<string>Downloads latest XLD disk image.</string>
+	<key>Identifier</key>
+	<string>com.github.autopkg.download.XLD</string>
+	<key>Input</key>
+	<dict>
+		<key>NAME</key>
+		<string>XLD</string>
+		<key>SPARKLE_FEED_URL</key>
+		<string>https://svn.code.sf.net/p/xld/code/appcast/xld-appcast_e.xml</string>
+	</dict>
+	<key>MinimumVersion</key>
+	<string>0.2.0</string>
+	<key>Process</key>
+	<array>
+		<dict>
+			<key>Arguments</key>
+			<dict>
+				<key>appcast_url</key>
+				<string>%SPARKLE_FEED_URL%</string>
+			</dict>
+			<key>Processor</key>
+			<string>SparkleUpdateInfoProvider</string>
+		</dict>
+		<dict>
+			<key>Arguments</key>
+			<dict>
+				<key>filename</key>
+				<string>%NAME%-%version%.dmg</string>
+			</dict>
+			<key>Processor</key>
+			<string>URLDownloader</string>
+		</dict>
+		<dict>
+			<key>Processor</key>
+			<string>EndOfCheckPhase</string>
+		</dict>
```

Comment:
No code signature verification? I'm not familiar with XLD, but is it signed?

Reply:
Not signed:

```
% codesign --deep -vv -r- /Volumes/XLD\ 20191004/XLD.app 
/Volumes/XLD 20191004/XLD.app: code object is not signed at all
In architecture: x86_64
```

Reply:
Ugh.